src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -240,7 +238,13 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
               </Form.Group>
             </Col>
           </Row>
-          <Row className="mx-2 justify-content-end">
+          {/* I don't think the scope, budget,and timeline can be removed from here without first updating the backend */}
```

Comment:
now that we're on a feature branch we can make breaking changes like this so it's fine to get rid of them